coderd/database/migrations/migrate_test.go

```diff
@@ -33,7 +33,7 @@ func TestMigrate(t *testing.T) {
 
 		db := testSQLDB(t)
 
-		err := database.MigrateUp(db)
+		err := migrations.MigrateUp(db)
```

Comment:
Since this package is called `migrations` now, we should change these function names to just be `Up`.